crates/topos-config/src/node.rs

```diff
@@ -21,29 +20,32 @@ pub enum NodeRole {
     FullNode,
 }
 
-#[derive(Serialize, Deserialize, Debug, Clone)]
-pub(crate) struct NodeConfig {
-    pub(crate) base: BaseConfig,
-    pub(crate) tce: Option<TceConfig>,
-    pub(crate) sequencer: Option<SequencerConfig>,
-    pub(crate) edge: Option<EdgeConfig>,
+#[derive(Serialize)]
+struct Noop {}
```

Comment:
It's a bit annoying to have to add this type. Wouldn't `()` work? If we indeed have to have this, a better name would be `NoopCommand` or something that explains what it is.